app/phpunit.xml.dist

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <!-- http://www.phpunit.de/manual/current/en/appendixes.configuration.html -->
```

Comment:
PHPUnit indicated that its XML config was deprecated, so I ran their built-in command `bin/phpunit --migrate-configuration` to update it